src/Web/View/Base/AbstractView.php

```
@@ -515,17 +574,81 @@ private function beforeRender(string $view, array $params): bool
+     * reusable template structures.
+     *
+     * @param string $viewFile Relative, absolute, or alias-based path to the view file.
+     * @param ViewContextInterface $context Context providing the base directory for relative resolution.
+     *
+     * @return string Resolved view file path, ready for normalization and extension handling.
+     *
+     * {@see Aliases} for path alias resolution.
+     * {@see ViewContextInterface} for the context contract.
      */
-    private function afterRender(string $view, string $content, array $params): void
+    private function determinePath(string $viewFile, ViewContextInterface $context): string
```

PHPStan (2.0.3)

Syntax error, unexpected T_PRIVATE on line 598


(phpstan.parse)